### PR TITLE
Janitorial changes

### PR DESCRIPTION
**Pull request by Inorien**
## What this does
- ~~Space Cleaner purges all reagents on a tile (base: it would just turn reagents into water)~~ See #24 
- Mop can be used on already-clean tiles to spill up to 20% of the mop contents on the floor (dropping to 1u below 5u on mop, then dumping the remainder at <1u), creating a new puddle of whatever was on the mop
- Changed mop melee sound to the whip sound
- Changed mop error text when attempting to use a dry mop to "The mop is dry!" (a la /vg/station13) rather than "The mop has no water!" as mops can be used with other things now

## Why it's good
Janitors will have their power back

## How it was tested
- Filled the mop with water, cleaned up some non-water puddles on the floor, clicked on some non-puddle floor
- Beat up some guy in the dev map with the mop

**Changelog**

- tweak: Mop can be used to make new puddles on already-clean floor
- tweak: Mop melee sound is the whip sound from /vg/station13 rather than MetalThud

### CODE REVIEW

File: Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs [hunk 1]
```diff
@@ -95,13 +96,50 @@ private void OnActivateInWorld(EntityUid uid, AbsorbentComponent component, User
...
+
+        if (TryComp<UseDelayComponent>(used, out var useDelay)
+            && _useDelay.IsDelayed((used, useDelay)))
+            return;
+
+        var solution = absorberSoln.Value.Comp.Solution;
+        var spillAmount = component.PickupAmount > solution.Volume ? solution.Volume : component.PickupAmount;
+
+        // spilling the entire 50u mop in one go felt weird, cap to 10u
+        spillAmount = spillAmount > 10 ? 10 : spillAmount;
```
Comment (spess-empyrean) [inline Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs:129]: How about a percentage of the mop's capacity

Comment (Inorien) [reply Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs:129]: could work, 20% with a minimum of 1u?

Comment (spess-empyrean) [reply Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs:129]: Sounds good to me

Comment (Inorien) [reply Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs]: mop contents c:
c > 5u : spill 20%
5u > c > 1u : spill 1u
1u > c : spill c

Comment (Inorien) [reply Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs]: behaves kinda funny at the end when theres very little left, water puddles evaporate fast but it makes a funny noise when you click it fast and i can see a clown or janitor having fun slapping the mop around

**Comment by Inorien** [comment]
Pending decision on if we want to keep the SS14 space cleaner implementation or switch to /vg/station13 behaviour

**Comment by Inorien** [comment]
moving spacecleaner change to separate PR

**Pull request merged**